dynamic/parameterize/args.go

```diff
@@ -31,27 +31,46 @@ type RemoteArgs struct {
 	Name string
 	// Version is the (possibly empty) version constraint on the provider.
 	Version string
+	// Docs is the (possibly empty) argument to download and write docs into the schema
+	Docs bool
 }
 
 // LocalArgs represents a local TF provider referenced by path.
 type LocalArgs struct {
 	// Path is the path to the provider binary. It can be relative or absolute.
 	Path string
+	// DocsLocation is the path to the provider documentation.
+	DocsLocation string
```

Comment:
Maybe we should call this `UpstreamRepoPath`, since that's what it is. 

In any case, its not the path to docs, but to the repo that contains the docs.

```suggestion
	// UpstreamRepoPath (if provided) is the local path to the dynamically bridged Terraform provider's repo.
	//
	// If set, full documentation will be generated for the provider. If not set, only documentation from the TF provider's schema will be used.
	UpstreamRepoPath string
```